src/barcodes/CODE128/CODE128.js

```diff
@@ -49,6 +53,15 @@ class CODE128 extends Barcode {
 		};
 	}
 
+	// GS1-128/EAN-128
+	encodeAsEan128() {
```

Comment:
Nice with a separate functions. Will make thing a lot cleaner 🙂 What do you think about naming it shouldEncodeAsEan128 or similar to clarify that it returns a boolean?